invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/SearchResultsBox.js

```diff
@@ -0,0 +1,75 @@
+// This file is part of Invenio
+// Copyright (C) 2024 CERN.
+//
+// Invenio is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { InvenioSearchPagination } from "@js/invenio_search_ui/components";
+import { i18next } from "@translations/invenio_requests/i18next";
```

Comment:
you shouldn't import translations from invneio_requests, the strings will not be marked for translations